### ISSUE TITLE
Docs for range are incorrect (duplicate of create)

### ISSUE THREAD

**Issue by paulreimer** [Reactive-Extensions/RxCpp#428]
Here:
http://reactive-extensions.github.io/RxCpp/namespacerxcpp_1_1sources.html#ab416d787c14569e1821e8bff4ef4cdd8

**Comment by kirkshoop** [comment]
yes, that looks like some simple typos here - https://github.com/Reactive-Extensions/RxCpp/blob/1b2e0589f19cb34d8cd58803677701dcf2161876/Rx/v2/src/rxcpp/sources/rx-range.hpp#L116..L142
